txn/txn.go

```diff
@@ -380,15 +381,23 @@ func (r *Runner) ChangeLog(logc *mgo.Collection) {
 // used during the normal operation of an application. Its purpose is to put
 // a system that has seen unavoidable corruption back in a working state.
 func (r *Runner) PurgeMissing(collections ...string) error {
-	type M map[string]interface{}
-	type S []interface{}
 
 	type TDoc struct {
-		Id       interface{} "_id"
-		TxnQueue []string    "txn-queue"
+		Id       interface{}   "_id"
+		TxnQueue []interface{} "txn-queue"
```

Comment:
For the record, I've now personally seen `nil` values in `txn-queue` fields at 2 customer sites now.